db/schema.rb

```diff
@@ -97,9 +97,9 @@
     t.boolean "tsf_primary_plus_eligibility", default: false
     t.enum "lead_provider_approval_status", enum_type: "lead_provider_approval_statuses"
     t.text "participant_outcome_state"
-    t.bigint "school_id"
```

Comment:
It doesn't look like these schemas should have changed?

Reply:
Yes, thanks. I had some changes from another branch.

Reply:
Addressed in https://github.com/DFE-Digital/npq-registration/pull/1435/commits/6c5d7bc408adb05bb25d669f610804bd78773b12